README.md

```diff
@@ -194,3 +218,25 @@ fix any issues it can, so you usually won't need to run it yourself.
 If you notice the linter doing unexpected things to your formatting, it's
 usually a sign that your original formatting wouldn't have been parsed as you
 intended it to be.
+
+[013]:
+  tools-and-technology/rfc-013-use-docker-to-deploy-and-run-applications-in-containers.md
```

Comment:
Looks like some of the linting is introducing line breaks here where we don't want them.

Reply:
Yeah, this was an autofix by the linting bot. I've disabled Prettier for the block of links so it's now happy!